### PR TITLE
feat(cubeSource): Add vtkCubeSource

### PR DESCRIPTION
**Pull request by luciemac**

### CODE REVIEW

File: Sources/Filters/Sources/CubeSource/index.js [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    polys[polyIndex++] = 4;
+    polys[polyIndex++] = 20;
+    polys[polyIndex++] = 21;
+    polys[polyIndex++] = 23;
+    polys[polyIndex] = 22;
+  }
+
+  publicAPI.setBounds = (xMin, xMax, yMin, yMax, zMin, zMax) => {
+    const bounds = [xMin, xMax, yMin, yMax, zMin, zMax];
+    model.setBounds(bounds);
```
Comment (jourdain) [inline Sources/Filters/Sources/CubeSource/index.js]: setBounds() should be called on publicAPI not on model.

**Comment by jourdain** [comment]
Minor fix, but looks good otherwise. A test could be nice too.

File: Sources/Filters/Sources/CubeSource/index.js (hunk 1 above)
Comment (luciemac) [inline Sources/Filters/Sources/CubeSource/index.js]: Actually, after double checking, this function cannot be reached. Only the second function setBounds(bounds) is. 
I think it is no longer necessary to keep it.

**Comment by luciemac** [comment]
I will a test!

File: Sources/Filters/Sources/CubeSource/index.js (hunk 1 above)
Comment (jourdain) [inline Sources/Filters/Sources/CubeSource/index.js]: You right, only one function can exist. But you can make it so both profiles can exist.

```
publicAPI.setBounds = (...bounds) => {
   let boundsArray = bounds;
   if (Arrays.isArray(bounds[0])) {
      boundsArray = bounds[0];
   }
   [...]
}
```

File: Sources/Filters/Sources/CubeSource/index.js [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+    polys[polyIndex++] = 4;
+    polys[polyIndex++] = 20;
+    polys[polyIndex++] = 21;
+    polys[polyIndex++] = 23;
+    polys[polyIndex] = 22;
+  }
+
+  publicAPI.setBounds = (...bounds) => {
+    let boundsArray = [];
```
Comment (jourdain) [inline Sources/Filters/Sources/CubeSource/index.js]: Could be written like below which prevent the else case.

```
let boundsArray = [].concat(bounds);
```

**Pull request merged**